extras/label_configs/carrots

```diff
@@ -1,5 +1,5 @@
 # Remove Github defaults
--import:clear-defaults
+-delete:all
```

Comment:
This is a little more dangerous than clear-defaults..

Do we need to change this?